payments-core/src/main/java/com/stripe/android/paymentsheet/injection/PaymentSheetViewModelModule.kt

```diff
@@ -131,5 +135,21 @@ internal abstract class PaymentSheetViewModelModule {
         @Singleton
         fun provideLogger(@Named(ENABLE_LOGGING) enableLogging: Boolean) =
             Logger.getInstance(enableLogging)
+
+        @Provides
+        @Singleton
+        fun provideAnalyticsRequestFactory(
+            appContext: Context,
+            lazyPaymentConfiguration: Lazy<PaymentConfiguration>
+        ) = AnalyticsRequestFactory(
+            appContext,
+            { lazyPaymentConfiguration.get().publishableKey },
+            setOf(PaymentSheetEvent.PRODUCT_USAGE)
+        )
+
+        @Provides
+        @Singleton
+        fun provideAnalyticsRequestExecutor(): AnalyticsRequestExecutor =
+            DefaultAnalyticsRequestExecutor()
```

Comment:
is this needed? seems we can just annotated `DefaultAnalyticsRequestExecutor` with `@Inject`

Reply:
How about moving it to `PaymentCommonModule`? that seems to be a place to provide deps from core

Reply:
Also regardless which module it's declared, I think it's better to mark `DefaultAnalyticsRequestExecutor`'s constructor as `@Inject` and do
```
@Binds
fun bindsAnalyticsRequestExecutor(val e: DefaultAnalyticsRequestExecutor): AnalyticsRequestExecutor

```